src/client/Microsoft.Identity.Client/Platforms/netdesktop/NetDesktopCryptographyManager.cs

```diff
@@ -18,49 +18,12 @@
 
 namespace Microsoft.Identity.Client.Platforms.net45
 {
-#if NET45
-    internal class NetDesktopCryptographyManager : ICryptographyManager
-#else
     internal class NetDesktopCryptographyManager : CommonCryptographyManager
-#endif
     {
         private static readonly ConcurrentDictionary<string, RSACryptoServiceProvider> s_certificateToRsaCspMap = new ConcurrentDictionary<string, RSACryptoServiceProvider>();
         private static readonly int s_maximumMapSize = 1000;
 
-#if NET45
-        public NetDesktopCryptographyManager(ILoggerAdapter logger = null) { }
-#else
-        public NetDesktopCryptographyManager(ILoggerAdapter logger = null) : base(logger) { }
-#endif
-#if NET45
-        public string CreateBase64UrlEncodedSha256Hash(string input)
-        {
-            return string.IsNullOrEmpty(input) ? null : Base64UrlHelpers.Encode(CreateSha256HashBytes(input));
-        }
-
-        public string GenerateCodeVerifier()
-        {
-            byte[] buffer = new byte[Constants.CodeVerifierByteSize];
-            using (var randomSource = RandomNumberGenerator.Create())
-            {
-                randomSource.GetBytes(buffer);
-            }
-
-            return Base64UrlHelpers.Encode(buffer);
-        }
-
-        public string CreateSha256Hash(string input)
-        {
-            return string.IsNullOrEmpty(input) ? null : Convert.ToBase64String(CreateSha256HashBytes(input));
-        }
-
-        public byte[] CreateSha256HashBytes(string input)
-        {
-            using (var sha = SHA256.Create())
-            {
-                return sha.ComputeHash(Encoding.UTF8.GetBytes(input));
-            }
-        }
+        public NetDesktopCryptographyManager(ILoggerAdapter logger = null) : base(logger) { }    
 
         public byte[] SignWithCertificate(string message, X509Certificate2 certificate)

```

Comment:
I think this can be removed too and the method in CommonCryptoManager can be used.
Also GetCryptoProviderForSha256_Net45 can be removed.

Or move GetCngPrivateKey and GetCertificateContext into CommonCryptoManager, and delete this class.

Reply:
Yes, deleted.